.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+addons:
+  apt:
+    packages:
+    - git
+    - make
+    - curl
+env:
+  - DOCKER_IMAGE_NAME=cloudposse/thumbor
```

Comment:
No. Because this is image name for travis ci.
Tag would be based on travis env vars. It would be new tag what will be used for push